lib/utilities.js

```diff
@@ -75,6 +76,20 @@ const buildOptions = function() {
     if (!options || typeof options !== 'object') return;
     Object.keys(options).forEach(i => result[i] = options[i]);
   });
+
+  // Ensure a hashAlgorithm option exists.
+  if (!result.hashAlgorithm) {
+    result.hashAlgorithm = 'md5';
+  }
+
+  // Ensure the configured hashAlgorithm is available on the system
+  if (crypto.getHashes().find(h => result.hashAlgorithm === h) === undefined) {
+    throw Error(
+      `Hashing algorithm '${result.hashAlgorithm}' is not supported by this system's OpenSSL ` +
+      `version`

```

Comment:
What is the purpose of such formatting with concats?

Reply:
I'm never sure if the + should be trailing, or start the new line. I moved to the new line for you. 

If the question is "why did I concatenate?": It exceeded the defined linter character limit.